src/Microsoft.TestPlatform.AdapterUtilities/Resources/Resources.Designer.cs

```diff
@@ -1,157 +1,163 @@
+            }
+        }
+        
         /// <summary>
         ///   Looks up a localized string similar to ManagedName is incomplete.
         /// </summary>
-        internal static string ErrorIncompleteManagedName
-        {
-            get
-            {
+        internal static string ErrorIncompleteManagedName {
+            get {
                 return ResourceManager.GetString("ErrorIncompleteManagedName", resourceCulture);
             }
         }
-
+        
+        /// <summary>
+        ///   Looks up a localized string similar to Invalid escape sequence! (segment: {0}, pos: {1}).
+        /// </summary>
+        internal static string ErrorInvalidSequenceAt {
+            get {
+                return ResourceManager.GetString("ErrorInvalidSequenceAt", resourceCulture);
+            }
+        }
+        
         /// <summary>
         ///   Looks up a localized string similar to Method arity must be numeric.
         /// </summary>
-        internal static string ErrorMethodArityMustBeNumeric
-        {
-            get
-            {
+        internal static string ErrorMethodArityMustBeNumeric {
+            get {
                 return ResourceManager.GetString("ErrorMethodArityMustBeNumeric", resourceCulture);
             }
         }
-
+        
+        /// <summary>
+        ///   Looks up a localized string similar to Argument must be a method..
+        /// </summary>
+        internal static string ErrorMethodExpectedAsAnArgument {
+            get {
+                return ResourceManager.GetString("ErrorMethodExpectedAsAnArgument", resourceCulture);
+            }
+        }
+        
         /// <summary>
         ///   Looks up a localized string similar to Method &apos;{0}&apos; not found on type &apos;{1}&apos;.
         /// </summary>
-        internal static string ErrorMethodNotFound
-        {
-            get
-            {
+        internal static string ErrorMethodNotFound {
+            get {
                 return ResourceManager.GetString("ErrorMethodNotFound", resourceCulture);
             }
         }
-
+        
+        /// <summary>
+        ///   Looks up a localized string similar to A closing single quote was expected at the end of the segment! (segment: {0}).
+        /// </summary>
+        internal static string ErrorNoClosingQuote {
+            get {
+                return ResourceManager.GetString("ErrorNoClosingQuote", resourceCulture);
+            }
+        }
+        
         /// <summary>
         ///   Looks up a localized string similar to Type &apos;{0}&apos; not found.
         /// </summary>
-        internal static string ErrorTypeNotFound
-        {
-            get
-            {
+        internal static string ErrorTypeNotFound {
+            get {
                 return ResourceManager.GetString("ErrorTypeNotFound", resourceCulture);
             }
         }
-
+        
```

Comment:
Can the generator no emit these whitespace empty lines?

Reply:
Idk, I'll check.